src/cryptography/falcon.rs

```diff
@@ -1,4 +1,13 @@
 pub mod inverse;
 pub mod cyclotomic_fourier;
 pub mod polynomial;
-pub mod field;
\ No newline at end of file
+pub mod field;
+pub mod fft;
+pub mod u32_field;
+pub mod sample;
+pub mod math;
+pub mod encoding;
+pub mod falcon;
+pub mod sampling;
+pub mod falcon512;
+pub mod falcon1024;
```

Comment:
The file should end with a NEW LINE.

```suggestion
pub mod falcon1024;

```

https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
I solved this